SupremacyBenchmark/privateBuild/Program.cs

```diff
@@ -0,0 +1,71 @@
+namespace quantum
```

Comment:
We might want to put some explanitory comments in the test harness before it gets posted publicly,